src/solvers/lowering/byte_operators.cpp

```diff
@@ -30,6 +31,7 @@ static exprt unpack_rec(
 /// \param src_size: array/vector size
 /// \param element_width: bit width of array/vector elements
 /// \param little_endian: true, iff assumed endianness is little-endian
+/// \param offset: if not nil, use as lower bound for non-nil bytes to return
```

Comment:
Odd phrasing! How about
`if not nil, bytes prior to this offset will be filled with nil values`

Reply:
Thank you :-)